### PR TITLE
[BUG] Repair JDBC connection of Oracle

### PR DESCRIPTION
**Pull request by jxeditor**
## Purpose of the pull request

Repair JDBC connection of Oracle

## Brief change log

Repair JDBC connection of Oracle
Oracle SID:
jdbc:oracle:thin:@<host>:<port>:<SID> 

Oracle Service Name
jdbc:oracle:thin:@//<host>:<port>/<service_name> 

## Verify this pull request

This pull request is code cleanup without any test coverage.

### CODE REVIEW

File: dolphinscheduler-datasource-plugin/dolphinscheduler-datasource-api/src/main/java/org/apache/dolphinscheduler/plugin/datasource/api/datasource/oracle/OracleDataSourceProcessor.java [hunk 1]
```diff
@@ -65,14 +65,16 @@ public BaseDataSourceParamDTO createDatasourceParamDTO(String connectionJson) {
...
         String address;
+        String jdbcUrl;
         if (DbConnectType.ORACLE_SID.equals(oracleParam.getConnectType())) {
             address = String.format("%s%s:%s",
                     Constants.JDBC_ORACLE_SID, oracleParam.getHost(), oracleParam.getPort());
+            jdbcUrl = address + "/" + oracleParam.getDatabase();
         } else {
             address = String.format("%s%s:%s",
                     Constants.JDBC_ORACLE_SERVICE_NAME, oracleParam.getHost(), oracleParam.getPort());
+            jdbcUrl = address + "/" + oracleParam.getDatabase();
```
Comment (zhuangchong) [inline dolphinscheduler-datasource-plugin/dolphinscheduler-datasource-api/src/main/java/org/apache/dolphinscheduler/plugin/datasource/api/datasource/oracle/OracleDataSourceProcessor.java]: I'm sorry, I didn't find the difference between the code you changed and before the change.

**Review by zhuangchong — CHANGES_REQUESTED** [review]

File: dolphinscheduler-datasource-plugin/dolphinscheduler-datasource-api/src/main/java/org/apache/dolphinscheduler/plugin/datasource/api/datasource/oracle/OracleDataSourceProcessor.java [hunk 2]
```diff
@@ -65,14 +65,16 @@ public BaseDataSourceParamDTO createDatasourceParamDTO(String connectionJson) {
     public BaseConnectionParam createConnectionParams(BaseDataSourceParamDTO datasourceParam) {
         OracleDataSourceParamDTO oracleParam = (OracleDataSourceParamDTO) datasourceParam;
         String address;
+        String jdbcUrl;
         if (DbConnectType.ORACLE_SID.equals(oracleParam.getConnectType())) {
             address = String.format("%s%s:%s",
                     Constants.JDBC_ORACLE_SID, oracleParam.getHost(), oracleParam.getPort());
+            jdbcUrl = address + "/" + oracleParam.getDatabase();
```
Comment (zhuangchong) [inline dolphinscheduler-datasource-plugin/dolphinscheduler-datasource-api/src/main/java/org/apache/dolphinscheduler/plugin/datasource/api/datasource/oracle/OracleDataSourceProcessor.java]: I understand what you might want to modify is this part.

SID:
```
jdbc:oracle:thin:@<host>:<port>:<SID> 
```
service_name:
```
jdbc:oracle:thin:@//<host>:<port>/<service_name> 

```

Comment (jxeditor) [reply dolphinscheduler-datasource-plugin/dolphinscheduler-datasource-api/src/main/java/org/apache/dolphinscheduler/plugin/datasource/api/datasource/oracle/OracleDataSourceProcessor.java]: yes,The previous code only supported servicename

**Review by jxeditor — COMMENTED** [review]
Sorry,No modification succeeded

File: dolphinscheduler-datasource-plugin/dolphinscheduler-datasource-api/src/main/java/org/apache/dolphinscheduler/plugin/datasource/api/datasource/oracle/OracleDataSourceProcessor.java (hunk 1 above)
Comment (jxeditor) [reply dolphinscheduler-datasource-plugin/dolphinscheduler-datasource-api/src/main/java/org/apache/dolphinscheduler/plugin/datasource/api/datasource/oracle/OracleDataSourceProcessor.java]: Aha,It didn't change successfully

**Review by zhuangchong — DISMISSED** [review]
+1

**Review by zhuangchong — APPROVED** [review]
+1

**Comment by zhuangchong** [comment]
considering that it's your first contribution, I think we can get deep communiction, you can contact me through mail or add wechat(KerwinZhuang), when mail or added, please tell me who you are, I think I can help to familiar with the DolphinScheduler if you meet with problems.

**Pull request merged**